source/backend/dal/Repositories/PropertyRepository.cs

```diff
@@ -56,25 +56,32 @@ public int Count()
         /// <returns></returns>
         public Paged<PimsPropertyVw> GetPage(PropertyFilter filter)
         {
-            this.User.ThrowIfNotAuthorized(Permissions.PropertyView);
+            User.ThrowIfNotAuthorized(Permissions.PropertyView);
             filter.ThrowIfNull(nameof(filter));
             if (!filter.IsValid())
             {
                 throw new ArgumentException("Argument must have a valid filter", nameof(filter));
             }
 
             var skip = (filter.Page - 1) * filter.Quantity;
-            var query = Context.GeneratePropertyQuery(this.User, filter);
+            var query = Context.GeneratePropertyQuery(User, filter);
             var items = query
                 .Skip(skip)
                 .Take(filter.Quantity)
                 .ToArray();
 
-            if (!string.IsNullOrWhiteSpace(filter.PinOrPid))
+            if (!string.IsNullOrWhiteSpace(filter.Pid))
+            {
+                Regex nonInteger = new Regex("[^\\d]");
+                var formattedPid = nonInteger.Replace(filter.Pid, string.Empty);
+                items = items.Where(i => i.Pid.ToString().PadLeft(9, '0').Contains(formattedPid)).ToArray();
+            }
+
+            if (!string.IsNullOrWhiteSpace(filter.Pin))
```

Comment:
is this necessary? it seems like prior to this there was some lpadding happening for the pid/pin, but it seems like a 2 part search isn't required for pin.

Reply:
I see your point - I will remove the 2-part search for PIN (and leave PID as-is)